### ISSUE TITLE
Steam OS 3.5 Preview, desktop mode extremely dark

### ISSUE THREAD

**Issue by rafrafek** [ValveSoftware/SteamOS#1157]
#### Your system information

* Steam client version: latest
* SteamOS version: latest Preview
* Have you checked for updates in Settings > System?: Yes

#### Please describe your issue in as much detail as possible:
Screen in Desktop mode is extremely dark

#### Steps for reproducing this issue:

1. Install latest Steam OS Preview version
2. Go to desktop mode
3. Screen is extremely dark (internal screen, I don't have any external screen to test with).
In game mode the screen is OK. It just affects desktop mode.

Similar old issue, not sure if related:
https://www.reddit.com/r/SteamDeck/comments/139s2h4/steam_os_35_desktop_mode_extremely_dark/

**Comment by steve228uk** [comment]
Did you previously dock or run with HDR? I had this same issue and needed to reboot to fix the colour space. It seems like that black level is incorrectly set when docking and undocking.

**Comment by rafrafek** [comment]
I used my steam deck with a usb-c 4k monitor two weeks ago, it was before switching from stable to preview.

Restarting helps but not for long. After 2-3 switches from game mode to desktop the darkness comes back.

**Comment by yuusharo** [comment]
> Did you previously dock or run with HDR? I had this same issue and needed to reboot to fix the colour space. It seems like that black level is incorrectly set when docking and undocking.

I have never docked my Steam Deck to a monitor at all, let alone an HDR one. I, too, have this issue randomly every 3-4 boots over to Desktop mode.

**Comment by BooBerry** [comment]
Yep, seeing this one as well. Even after a clean install of 3.5 preview, it never happened on stable.

It's seemingly random.

**Comment by octopuserectus** [comment]
This was also an issue in late April when the Saturation slider was first added on Main branch.

**Comment by SorceressLyra** [comment]
I'm experiencing the same issue, dark on desktop mode normal in game-mode.

After a reboot desktop mode might be normal, but usually goes dark again after switching to game-mode and back.

**Comment by Rephlexion** [comment]
On Preview 3.5: While streaming from my Windows 11 PC, the Steam Deck's display is very dark (indicative of low gamma) except when pressing the volume buttons or the Steam / quick access menu buttons, which temporarily brightens up the entire screen while an overlay element is active (but the screen appears to be very washed out, indicative of high gamma)

Yesterday during at least one session, I was able to get the screen to brighten up for longer than a few seconds if I moved a menu cursor to the right or left side of the screen, but that no longer seems to work.

**Comment by HugLifeTiZ** [comment]
I ran into this at first, too. I used the 2.0 gamma fix in desktop mode to correct it. But when I changed the color temperature in game mode to 6500K, it went away, and I had to revert the gamma change. I don't know for sure if these two things are related.

**Comment by eth-p** [comment]
This appears to have been an issue for a while, unfortunately. https://steamcommunity.com/app/1675200/discussions/1/3824162051004262723/

If you want to save a few clicks with the gamma workaround, I've been using a `.desktop` file on my desktop.

```
[Desktop Entry]
Exec=xrandr --output eDP --gamma 2:2:2
Icon=preferences-desktop-screensaver-symbolic
Name=Fix Gamma (Built-in)
StartupNotify=false
Terminal=false
TerminalOptions=
Type=Application
```

---

@Rephlexion Just curious, but if you disable hardware decoding for Remote Play, does that fix the brightness? I've been trying to find a better workaround than disabling the hardware decoder, but no luck yet.

**Comment by Rephlexion** [comment]
> This appears to have been an issue for a while, unfortunately. https://steamcommunity.com/app/1675200/discussions/1/3824162051004262723/
> 
> If you want to save a few clicks with the gamma workaround, I've been using a `.desktop` file on my desktop.
> 
> ```
> [Desktop Entry]
> Exec=xrandr --output eDP --gamma 2:2:2
> Icon=preferences-desktop-screensaver-symbolic
> Name=Fix Gamma (Built-in)
> StartupNotify=false
> Terminal=false
> TerminalOptions=
> Type=Application
> ```
> 
> @Rephlexion Just curious, but if you disable hardware decoding for Remote Play, does that fix the brightness? I've been trying to find a better workaround than disabling the hardware decoder, but no luck yet.

Hey, sorry for the delay and thanks for the .desktop file! I can't test it at the moment, but I do believe disabling Hardware Decoding does fix these gamma problems.

**Comment by SorceressLyra** [comment]
This issue hasn't appeared for me since the latest preview update.

**Comment by yuusharo** [comment]
> This issue hasn't appeared for me since the latest preview update.

I can confirm that the latest preview update of SteamOS 3.5.1 plus the most current flathub updates for KDE and Mesa still have this issue.

It seems to happen less frequently, but I was hit by the gamma issue once yesterday. Still not resolved unfortunately.

**Comment by SorceressLyra** [comment]
> > This issue hasn't appeared for me since the latest preview update.
> 
> I can confirm that the latest preview update of SteamOS 3.5.1 plus the most current flathub updates for KDE and Mesa still have this issue.
> 
> It seems to happen less frequently, but I was hit by the gamma issue once yesterday. Still not resolved unfortunately.

Yeah I can confirm it's happening to me again.

**Comment by BernalCarlos** [comment]
I'm having the same problem

**Comment by BernalCarlos** [comment]
> This appears to have been an issue for a while, unfortunately. https://steamcommunity.com/app/1675200/discussions/1/3824162051004262723/
> 
> If you want to save a few clicks with the gamma workaround, I've been using a `.desktop` file on my desktop.
> 
> ```
> [Desktop Entry]
> Exec=xrandr --output eDP --gamma 2:2:2
> Icon=preferences-desktop-screensaver-symbolic
> Name=Fix Gamma (Built-in)
> StartupNotify=false
> Terminal=false
> TerminalOptions=
> Type=Application
> ```
> 
> ---
> 
> @Rephlexion Just curious, but if you disable hardware decoding for Remote Play, does that fix the brightness? I've been trying to find a better workaround than disabling the hardware decoder, but no luck yet.
> 
> 

@eth-p Disabling hardware decoding on the  remote play client settings, solved the issue for me.

**Comment by krzysztofglab** [comment]
> > Replying to [#1157 (comment)](https://github.com/ValveSoftware/SteamOS/issues/1157#issuecomment-1758904517)
> 
> @eth-p Disabling hardware decoding on the remote play client settings, solved the issue for me.

Yes, but if you disable hardware decoding, you're no longer able to enter steam/performance quick settings. They should finally fix this.

**Comment by weeurey** [comment]
I'm still having this issue.

**Comment by weeurey** [comment]
Using the non preview release branch too

**Comment by gcaragea** [comment]
Happening consistently for me, remote play with hardware decoding is very dark. Switching to software decoding fixes the problem, but it increases latency and also disables HEVC, so it's a double bummer.

**Comment by Mateok51** [comment]
I am also having this issue. Steam Deck is unusable while docked. Is it possible that this issue will never be fixed? Buying the dock was a waste of money
